projectroles/forms.py

```diff
@@ -957,6 +957,29 @@ def clean(self):
         except ProjectInvite.DoesNotExist:
             pass
 
+        # Local users check
+        try:
+            user_email = self.cleaned_data.get('email')
+            domain = (
+                user_email[user_email.find('@') + 1 :].split('.')[0].lower()
+            )
+            domain_list = getattr(settings, 'LDAP_ALT_DOMAINS', []) + [
+                getattr(settings, 'AUTH_LDAP_USERNAME_DOMAIN', ''),
+                getattr(settings, 'AUTH_LDAP2_USERNAME_DOMAIN', ''),
+            ]
+            if (
+                not settings.PROJECTROLES_ALLOW_LOCAL_USERS
+                and not settings.ENABLE_SAML
+                and not any(domain in d.lower() for d in domain_list)
```

Comment:
As stated before, we need to compare the values in `LDAP_ALT_DOMAINS` to the entire domain of the recipient email containing the TLD. Right now we split the domain to not include the TLD and use that for all comparisons. This is going to need two conditions in the if statement for `AUTH_LDAP*_USERNAME_DOMAIN` (without TLD) and `LDAP_ALT_DOMAINS` (with TLD).